doc/installing-windows.md

```diff
@@ -0,0 +1,79 @@
+# Using hipSYCL on Windows
```

Comment:
As this is more of an experimental guide rather than "official documentation", I wonder if the project Wiki might be a more appropriate place for it as it might allow for quicker editing and adapting to changes? We could still link to it in the main readme.